modgui/icon-velocitygamma.html

```diff
@@ -0,0 +1,66 @@
+<div class="mod-pedal x42-midifilter{{{cns}}} x42-midi-velocitygamma">
+    <div mod-role="drag-handle" class="mod-drag-handle"></div>
+    <div class="x42-brand"><h1 title="Brought to you by x42.">x42</h1></div>
+    <div class="x42-plugin-name"><h1>MIDI Velocity Gamma</h1></div>
+
+    <div class="mod-light on" mod-role="bypass-light"></div>
+    <div class="mod-footswitch" mod-role="bypass"></div>
+
+    <div class="mod-control-group mod-enumerated-group bottom clearfix">
+        <span class="mod-knob-title">Midi Channel</span>
+        <div class="mod-enumerated" mod-role="input-control-port" mod-port-symbol="channel" mod-widget="custom-select">
+          <div mod-role="input-control-value" mod-port-symbol="channel" class="mod-enumerated-selected"></div>
+          <div class="mod-enumerated-list">
+            <div mod-role="enumeration-option" mod-port-value="0">Any</div>
+            <div mod-role="enumeration-option" mod-port-value="1">1</div>
+            <div mod-role="enumeration-option" mod-port-value="2">2</div>
+            <div mod-role="enumeration-option" mod-port-value="3">3</div>
+            <div mod-role="enumeration-option" mod-port-value="4">4</div>
+            <div mod-role="enumeration-option" mod-port-value="5">5</div>
+            <div mod-role="enumeration-option" mod-port-value="6">6</div>
+            <div mod-role="enumeration-option" mod-port-value="7">7</div>
+            <div mod-role="enumeration-option" mod-port-value="8">8</div>
+            <div mod-role="enumeration-option" mod-port-value="9">9</div>
+            <div mod-role="enumeration-option" mod-port-value="10">10</div>
+            <div mod-role="enumeration-option" mod-port-value="11">11</div>
+            <div mod-role="enumeration-option" mod-port-value="12">12</div>
+            <div mod-role="enumeration-option" mod-port-value="13">13</div>
+            <div mod-role="enumeration-option" mod-port-value="14">14</div>
+            <div mod-role="enumeration-option" mod-port-value="15">15</div>
+            <div mod-role="enumeration-option" mod-port-value="16">16</div>
+          </div>
+        </div>
+    </div>
+
+
+    <div class="mod-control-group top clearfix">
+        <div class="mod-knob">
+          <span class="mod-knob-title">Note-on Gamma</span>
+            <div class="mod-knob-n127-image" mod-role="input-control-port" mod-port-symbol="ongamma"></div>
+        </div>
+    </div>
+
+    <div class="mod-control-group top clearfix">
+        <div class="mod-knob">
+          <span class="mod-knob-title">Note-off Gamma</span>
+            <div class="mod-knob-n127-image" mod-role="input-control-port" mod-port-symbol="ongamma"></div>
```

Comment:
This should control `offgamma`, and `use mod-knob-n128-image` (note-off can include zero, 0..127, 128 possible note values).

In any case I think mapping this to a knob is preferred. Shoing LEDs in the range 1..127, 0..127 for values [0..5] is confusing